app/models/review.rb

```diff
@@ -0,0 +1,29 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :film
+  has_many :comments, as: :commentable
+  has_many :ratings, as: :rateable
+
+  validates :review_title, :review_content, :user_id, :film_id, { presence: true }
+
+  def total_ratings
+    if ratings.any?
```

Comment:
If more than one model has "ratings" behavior, pull it out into a ruby module or an ActiveSupport::Concern